tests/task/test_training.py

```diff
@@ -3,12 +3,13 @@
 
 import pytest
 import sentencepiece as spm
-from fixtures import DataDir, en_sample, ru_sample, zh_sample, FIXTURES_PATH
+from pathlib import Path
+from tests.fixtures import DataDir, en_sample, ru_sample, zh_sample, FIXTURES_PATH
 
 pytestmark = [pytest.mark.docker_amd64]
 
 current_folder = os.path.dirname(os.path.abspath(__file__))
-fixtures_path = os.path.join(current_folder, "fixtures")
+fixtures_path = (Path(__file__).resolve().parents[1] / "fixtures").as_posix()
```

Comment:
Same here with my previous comments on pathlib. I'll stop commenting on this, but if you can look through the rest of the PR for anything else where my feedback would apply.